pycomlink/processing/k_R_relation.py

```diff
@@ -183,32 +185,70 @@ def a_b(f_GHz, pol, approx_type="ITU_2005"):
         prediction methods", International Telecommunication Union, P.838-2 (04/2003) P.838-3 (03/2005)
 
     """
-    from scipy.interpolate import interp1d
+    if isinstance(f_GHz, xr.DataArray):
+        return_xarray = True
+        f_GHz_coords = f_GHz.coords
+    else:
+        return_xarray = False
+
+    f_GHz = xr.DataArray(f_GHz)
```

Comment:
We allow certain combinations of np.ndarray, xr.DataArray and float or str. But other combinations are not allowed. I suggest to add that info to the docstring at `f_GHz` and `pol`.

I know that we do not mention xr.DataArray as input data type in many docstrings, even though they are handled via the xarray-wrapper decorator. I actually always wanted to add the featuer that this wrappers add a note about that behavior to the docstring of the wrapped function. Yet another todo... But here, since we actually handled xr.DataArrays expliclity in the function body, I would adjust the docstring accordingly.